exonum-java-binding-core/src/test/java/com/exonum/binding/blockchain/BlockchainTest.java

```diff
@@ -66,4 +67,11 @@ void getBlockTransactions() {
     assertThat(blockchain.getBlockTransactions(1L)).isEqualTo(mockListIndex);
   }
 
+  @Test
+  void getActualConfiguration() {
+    StoredConfiguration configuration = mock(StoredConfiguration.class);
```

Comment:
It's usually not recommended to mock values (rather instantiate them), but I do not insist.

As a side note, don't you think we will benefit from using a builder: https://github.com/google/auto/blob/master/value/userguide/builders.md? If so, I'd add a task to jira.

Reply:
Builder won't hurt definitely.